### ISSUE TITLE
Throws screen_brightness_control.ScreenBrightnessError

### ISSUE THREAD

**Issue by patwadeepak** [Crozzers/screen_brightness_control#11]
I got following error when ```sbc.windows.get_display_info()```  is called.
```
screen_brightness_control.ScreenBrightnessError:
        filter_monitors -> TypeError: string indices must be integers
```
The issue originates from ```sbc.windows.list_monitors_info()``` assuming that ```sbc.windows.get_display_info()``` will return a valid list of dictionaries but it does not.

It returned a list like  ```[{...}, 'string']``` instead in my case.

**Comment by Crozzers** [comment]
This issue was patched with v0.11.2 but I am still not sure why the issue happened. If you could send the output of `sbc.windows.get_display_info()` then it might help me understand why this issue appeared

**Comment by Crozzers** [comment]
Closed due to inactivity
